code-studio/src/js/components/AssetRow.jsx

```diff
@@ -7,7 +7,7 @@ var AssetThumbnail = require('./AssetThumbnail');
 var AssetRow = React.createClass({
   propTypes: {
     name: React.PropTypes.string.isRequired,
-    type: React.PropTypes.oneOf(['image', 'audio', 'video']).isRequired,
+    type: React.PropTypes.oneOf(['image', 'audio', 'video', 'pdf']).isRequired,
```

Comment:
This is the list of valid icons to show — the filtering happens in `allowedExtensions`.